coq-primitive.opam

```diff
@@ -0,0 +1,31 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+version: "dev"
+synopsis: "The Coq Proof Assistant -- Primitive Objects"
+description: """
+This package provides OCaml implementations of Coq's primitive
+objects, used by coq-core and also available to link with extracted code."""
+maintainer: ["The Coq development team <coqdev@inria.fr>"]
+authors: ["The Coq development team, INRIA, CNRS, and contributors"]
+license: "LGPL-2.1-only"
```

Comment:
I thought that we had discussed making this package MIT licensed (and that you had collected all the necessary authorizations from the relevant contributors). If that's the case, then the headers of the files contained in the packages should be adapted as well.